mathesar_ui/src/pages/schemas/Schemas.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import {
+    faPlus,
+    faTrash,
+    faPencilAlt,
+  } from '@fortawesome/free-solid-svg-icons';
+
+  import { schemas } from '@mathesar/stores/schemas';
+  import type { SchemaMap } from '@mathesar/stores/schemas';
+  import {
+    Icon,
+    Button,
+    SchemaRow,
+    TextInput,
+  } from '@mathesar-components';
+
+  // Prop
+  export let database: string;
+
+  let filterQuery = '';
+  const schemaList = [...($schemas.schemaMap as SchemaMap).values()];
```

Comment:
`$schemas.data` would return an array of the schemas. It can be used used directly.

The map is created additionally while fetching the schemas as a way of accessing data of particular schemas in O(1) time.

Reply:
Updated to use `$schemas.data` in 40096f4.